modules/gapi/src/executor/gexecutor.cpp

```diff
@@ -202,14 +286,20 @@ void cv::gimpl::GExecutor::run(cv::gimpl::GRuntimeArgs &&args)
                 auto& out_mat = *get<cv::Mat*>(args.outObjs.at(index));
                 createMat(desc, out_mat);
             }
-            // In the case of own::Mat never reallocated, checked to perfectly fit required meta
+            // In the case of own::RMat check to fit required meta
             else
```

Comment:
BTW Does this mean that `cv::UMat` is not supported as output arguments ?

Reply:
Yes, you're right. Frankly I didn't find any tests with UMat as computation input/output, so I'm not sure if it was supported before